src/app/space/create/pipelines/services/pipelines.service.ts

```diff
@@ -24,6 +24,10 @@ import { AuthenticationService } from 'ngx-login-client';
 
 import { NotificationsService } from 'app/shared/notifications.service';
 
+import { BuildConfig } from 'a-runtime-console/index';
```

Comment:
Do you think we need to bring this into the main app? Out of runtime-console.

Reply:
Hmm good question. I would ultimately like to bring the entire pipelines work out of runtime-console so we can delete the runtime-console folder. In the process, I would like to refactor it to be simpler as well.

I'm not sure how much benefit there will be to moving this specific export out of the runtime console; where it lives doesn't change much as far as I understand it.